### ISSUE TITLE
Responsive tables don't wrap cell contents in IE11

### ISSUE THREAD

**Issue by benjimarshall** [nhsuk/nhsuk-frontend#712]
# Bug Report

## What is the issue?

Responsive table cell contents do not wrap in IE11, unlike other browsers, when in the narrow mobile-mode. This can mean that horizontal scrolling is required, which is not ideal. This was noticed on my project when testing in IE11 for a small desktop (1280x1024) at high zoom.

## What steps are required to reproduce the issue?

This is only an issue for the responsive, >3 column tables. Using one of these tables, and IE11, reduce browser width or increase zoom until the mobile version of the table is seen. Eg. for a large desktop (1080px width) this is at 250% zoom, for a small desktop (1280px width) it is 170% zoom. Text will wrap as needed when in the table is in its desktop configuration, with multiple columns, and stop wrapping when in the mobile mode with one cell per visual row. 

As far as I can tell this occurs with every responsive table, and is an issue with the fact that each table cell is a flexbox in the mobile mode, with the cell heading and contents as flex items.

## What was the environment where this issue occurred?

Include the following as a minimum e.g.
* Device: Dell XPS 15
* Operating System: Windows 10 (Version 20H2, OS Build 19042.804)
* Browser: Internet Explorer 11
* Browser version: 9.11.19041.0
* NHS.UK frontend package version: 4.1.0
* Node version: v15.9.0
* npm version: 7.5.4

## Is there anything else you think would be useful in recreating the issue?

This can be seen in the [responsive table example](https://service-manual.nhs.uk/design-example/table-responsive), at 500% zoom and a small desktop screen (1280x1024):
### In Internet Explorer 11, cell contents don't wrap
![ie11_responsive_table_zoom500_width1280](https://user-images.githubusercontent.com/3650110/110155113-0af4ec80-7ddd-11eb-9a75-525c63ec9a68.png)
### In Firefox 86 cell contents wrap nicely. Chrome looks the same
![firefox_responsive_table_zoom500_width1280](https://user-images.githubusercontent.com/3650110/110156610-1ba66200-7ddf-11eb-9b8e-0c8933794d67.png)

This example is not ideal, but could be worse. On my project this is causing issues for cells that contain an NHS trust name, because some trust names are rather long. For us the issues start at >300% zoom on a small desktop.

**Comment by Fenwick17** [comment]
Hi @benjimarshall thanks for raising this issue, I will investigate it. I will post back any findings.

**Comment by Fenwick17** [comment]
@benjimarshall One thing that I have found which works to allow the contents to wrap would be by modifying the `td` to be block for IE only such as:
```
@media all and (-ms-high-contrast: none) {
  display: block;
}
```
This would not give the exact same appearance you have with Firefox and other modern browsers, but would still be functional and headings/data would still show.
![image](https://user-images.githubusercontent.com/11258920/110325758-a40b4980-800f-11eb-91d2-2cefb1b3f34d.png)
I will do some further investigation on this to see if we can easily get IE11 to behave the same as the other browsers. But for now you could add that piece of CSS in which should resolve the issue.

**Comment by benjimarshall** [comment]
@Fenwick17 thanks for that! I'll have a go with that bit of CSS

**Comment by AdamChrimes** [comment]
This will be fixed as part of v5.0.0, which we are hoping to release early next week.

#714
